microsetta_interface/implementation.py

```diff
@@ -39,9 +40,14 @@ class Source:
     SOURCE_TYPE_ENVIRONMENT = "environmental"
 
 
-PUB_KEY = pkg_resources.read_text(
-    'microsetta_interface',
-    "authrocket.pubkey")
+PUBKEY_ENVVAR = 'MICROSETTA_INTERFACE_DEBUG_JWT_PUB'
+if os.environ.get(PUBKEY_ENVVAR, False):
```

Comment:
Why environment variable over the config file?

Reply:
This is special case, and it would be nice to make it hard for production to see this stuff. But if you'd prefer it in server_config then we can do that

Reply:
I'm going to keep as is, we can revise in a future refactor if needed